presc/training_stability/cross_validation.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+from math import ceil
+
+from sklearn.model_selection import cross_validate
+from sklearn.model_selection import KFold
+
+
+def explore_cross_validation_kfolds(
+    dataset,
+    pipeline,
+    metrics=["accuracy"],
+    kfolds_list=[3, 4, 5, 7, 10],
+    repetitions=1,
+    minimum_repetitions=False,
+):
+    """Explore the cross-validation k-fold number with the specified pipeline.
+
+    Choosing the number of k-folds with which to carry out the cross-validation
```

Comment:
I see you've got these paragraphs in the notebook as well, and I think they belong there rather than here. Perhaps replace with a short description of what this function does.